### PR TITLE
add twenty-tinybird package

### PR DESCRIPTION
**Pull request by anamarn**
https://github.com/twentyhq/private-issues/issues/75

**TLDR**

Add twenty-server package, wich contains the last tinybird datasources and pipes used in analytics. 

This new version of the API endpoints, pipes and datasources are inspired by the implementation of dub.co (https://github.com/dubinc/dub/blob/main/packages/tinybird/). 

It contains the webhooks analytics, serverless functions duration and serverless functions error count analytics. As well as 

**In order to test**

- Follow the instructions in the README.md on twenty-tinybird using your admin token from twenty_analytics_playground if you want to modify them.
- For a better experience add the extension Tinybird support for VsCode 
- If you want more info about datasources and pipes please read the tinybird docs.

### CODE REVIEW

File: packages/twenty-tinybird/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+## How to use
+Create a virtual enviroment and install tinybird
+```sh
+python3 -m venv .venv
+source .venv/bin/activate
```
Comment (FelixMalfait) [inline packages/twenty-tinybird/README.md:6]: add .venv to .gitignore then :)

File: packages/twenty-tinybird/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+** List of available regions:
+[1] us-east4 (gcp) (https://app.us-east.tinybird.co)
+[2] europe-west3 (gcp) (https://app.tinybird.co/gcp/europe-west3)
+[3] us-east-1 (aws) (https://app.tinybird.co/aws/us-east-1)
+[4] us-west-2 (aws) (https://app.tinybird.co/aws/us-west-2)
+[5] eu-central-1 (aws) (https://app.tinybird.co/aws/eu-central-1) <- this
+[0] Cancel
+
+Use region [1]:
```
Comment (FelixMalfait) [inline packages/twenty-tinybird/README.md:21]: Region 5 no?

File: .github/workflows/ci-tinybird.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  pull_request:
+    
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  ci:
+    uses: tinybirdco/ci/.github/workflows/ci.yml@main
```
Comment (charlesBochet) [inline .github/workflows/ci-tinybird.yaml:16]: add step to check changed files

File: .github/workflows/ci-tinybird.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    with:
+        data_project_dir: packages/twenty-tinybird
+    secrets:
+        tb_admin_token: ${{ secrets.TINYBIRD_ADMIN_TOKEN }}
```
Comment (charlesBochet) [inline .github/workflows/ci-tinybird.yaml:20]: is this a fake token? or pointing to a test account?

File: packages/twenty-tinybird/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+```sh
+tb push --force --push-deps
+```
+To pull data from Tinybird use:
+```sh
+tb pull
+```
+Things I learned:
+
+When creating a MV think about populating it first using the files from the fixtures
```
Comment (charlesBochet) [inline packages/twenty-tinybird/README.md:36]: MV?

Comment (charlesBochet) [reply packages/twenty-tinybird/README.md:36]: add bullet points for lisibility?

**Review by charlesBochet — APPROVED** [review]
